_episodes/05-r-telemetry-report-array.md

```diff
@@ -9,7 +9,7 @@ questions:
 
 ### Mapping GLATOS stations - Static map
 
-This section will use the Receivers CSV for the entire GLATOS Network.
+This section will use the Receivers CSV for the entire ACT Network.
```

Comment:
this is not correct, we are using a subset based on approved projects for public consumption, consider rewording

Reply:
duplicate of below